Makefile

```diff
@@ -37,9 +36,17 @@ install-bin: bin/kind bin/kubectl-kuttl
 .PHONY: create-kind-cluster
 create-kind-cluster: $(KUBECONFIG)
 
+kubeaddons-tests:
+	git clone --depth 1 https://github.com/mesosphere/kubeaddons-tests.git --branch master --single-branch
+
 $(KUBECONFIG): install-bin
-	bin/kind create cluster --wait 10s --config=test/kind/$(KUBERNETES_VERSION).yaml
+	bin/kind create cluster --wait 10s --image=kindest/node:v$(KUBERNETES_VERSION)
 
 .PHONY: kind-test
-kind-test: create-kind-cluster
-# TODO restore once catalog is tested
+kind-test: kubeaddons-tests create-kind-cluster
```

Comment:
Not yours, but dependency resolution might not work as expected here. If a kind cluster already exists, running `make kind-test` will run `create-kind-cluster` again, probably resulting in errors. If only the `kind-test` target is expected to be run regularly by this Makefile, a batch script would be a better approach.
If other targets are used a lot, we should have targets create files that indicate the current state.